components/webgpu/lib.rs

```diff
@@ -457,10 +468,37 @@ impl WGPU {
                     let _ = gfx_select!(program_id =>
                         global.device_create_shader_module(device_id, &descriptor, program_id));
                 },
+                WebGPURequest::CreateTexture {
+                    device_id,
+                    texture_id,
+                    descriptor,
+                } => {
+                    let global = &self.global;
+                    let st = CString::new(descriptor.label.as_bytes()).unwrap();
```

Comment:
that should probably be changed in a way that doesn't allocate `CString` if the label is not given, but we can follow up with that

Reply:
Code related to labels needs to be updated on the client-side as well- use labels provided in the descriptor and update the webidl too. So, as you suggested, I'll handle all this in a follow-up PR.